letslambda.py

```diff
@@ -1,30 +1,25 @@
 # -*- coding: utf-8 -*-
 
-import yaml
-from Crypto.PublicKey import RSA
-from boto import s3
-from boto import iam
-from boto import route53
-from boto.s3.connection import OrdinaryCallingFormat
-from boto.s3.key import Key
-from boto.exception import S3ResponseError
+import base64
+import boto3
+import hashlib
 import logging
+import os
+import requests
+import yaml
+from acme import challenges
 from acme import client
+from acme import errors
 from acme import messages
-from acme import challenges
-import io
-import requests
-import base64
-import hashlib
-from time import sleep
-from OpenSSL import crypto
-from datetime import datetime
-from boto.ec2 import elb
 from acme.jose.util import ComparableX509
-import os
-
-# this is the AWS region in which the function runs in
-exec_region = os.environ['AWS_DEFAULT_REGION']
+from botocore.config import Config
+from botocore.exceptions import ClientError
+from Crypto import Random
+from Crypto.Cipher import AES
```

Comment:
I am unsure where `AES` is used in the line you are mentioning. Can you provide more details ?